src/Utils.ts

```diff
@@ -1,5 +1,9 @@
 export type NonEmptyArray<T> = [T, ...T[]];
 
+export function foldMapNonEmpty<T, R>(xs: NonEmptyArray<T>, f: (t: T) => R, g: (current: R, acc: R) => R) {
```

Comment:
This is missing a return type as noted by the linter. Can you lint and fix the errors? I fixed some lint errors on `master`, so you might want to rebase.